src/libusermode/userhook.cpp

```diff
@@ -286,7 +286,7 @@ static dll_t* create_dll_meta(drakvuf_t drakvuf, drakvuf_trap_info* info, userho
 
 static bool make_trap(vmi_instance_t vmi, drakvuf_t drakvuf, drakvuf_trap_info* info, hook_target_entry_t* target, addr_t exec_func)
 {
-    target->pid = info->proc_data.pid;
+    target->pid = info->attached_proc_data.pid;
```

Comment:
I think this needs to be OS agnostic. So at least check whether the target is Windows and only then use attached_proc_data.

Reply:
My fault! I will fix this. Thank You.

Reply:
Done